site/app/controllers/student/SubmissionController.php

```diff
@@ -486,6 +486,22 @@ public function ajaxBulkUpload($gradeable_id) {
 
                 $bulk_upload_job  = "/var/local/submitty/daemon_job_queue/bulk_upload_" . $uploaded_file["name"][$i] . ".json";
 
+                // exec() and similar functions are disabled by security policy,
+                // so we are using a python script via CGI to validate whether file is divisible by num_page or not.
+                $pdf_full_path = $pdf_path . "/" . $job_data["timestamp"] . "/" . $job_data["filename"];
```

Comment:
use FileUtils::joinPaths instead I think

Reply:
Okay. I will make this change and push the changes as soon as possible.

Reply:
Done. @04vmatsibekker